semantics/language/translation/decl/tagged.k

```diff
@@ -27,7 +27,8 @@ module C-DECL-TAGGED
           => addTag(X, makeStructFieldInfo(L))
           requires L =/=K .List
           [structural]
-     rule (.K => UNDEF("CDT1", "structs cannot be empty.", "6.7.2.1") )
+    //Undefinedness occurs when there are no named members in a struct. An empty struct is a contraint Violation. 
+     rule (.K => CV("CDT1", "structs cannot be empty.", "6.7.2.1") )
```

Comment:
Actually hang on. I am looking at the line in more detail and it does not seem to actually be a constraint violation as far as I can tell. Manasvi, can you tell me the paragraph you are looking at that you think is the constraint? If it's paragraph 2, you are misreading it like I originally did.